src/Models/SchemaOrg/EngineSpecification.php

```diff
@@ -7,32 +7,40 @@
  */
 class EngineSpecification extends \OpenActive\Models\SchemaOrg\StructuredValue
 {
+    /**
+     * @return string[]|null
+     */
+    static public function getType()
+    {
+        return "schema:EngineSpecification";
+    }
+
     /**
      * The type of fuel suitable for the engine or engines of the vehicle. If the vehicle has only one engine, this property can be attached directly to the vehicle.
      *
      *
-     * @var string|Schema.NET.QualitativeValue|null
+     * @var Schema.NET.QualitativeValue|string|null
      */
     protected $fuelType;
 
     /**
-     * @return string|Schema.NET.QualitativeValue|null
+     * @return Schema.NET.QualitativeValue|string|null
      */
     public function getFuelType()
     {
         return $this->fuelType;
     }
 
     /**
-     * @param string|Schema.NET.QualitativeValue|null $fuelType
+     * @param Schema.NET.QualitativeValue|string|null $fuelType
      * @return void
      * @throws \OpenActive\Exceptions\InvalidArgumentException If the provided argument is not of a supported type.
      */
     public function setFuelType($fuelType)
     {
         $types = array(
-            "string",
             "Schema.NET.QualitativeValue",
```

Comment:
Should this be `\OpenActive\Enums\SchemaOrg\QuantitativeValue` instead? Likely an issue in `models-lib` repo.

Reply:
Yup, as we saw together last night this is an issue with the generator so will ignore for the purposes of this PR but have noted the issue to fix later.